hack/ci/e2e-k8s.sh

```diff
@@ -161,6 +181,10 @@ EOF
     --wait=1m \
     -v=3 \
     "--config=${ARTIFACTS}/kind-config.yaml"
+
+  # Patch kube-proxy to set the verbosity level
+  kubectl patch -n kube-system daemonset/kube-proxy \
+    --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/command/3", "value": "--v='${KIND_CLUSTER_LOG_LEVEL}'" }]'
```

Comment:
`/spec/template/spec/containers/0/command/3`
->
`/spec/template/spec/containers/0/command/-`
to add at the end